benchmarks/tsmap_estimator.py

```diff
@@ -0,0 +1,94 @@
+import time
+import astropy.units as u
+import numpy as np
+import yaml
+from pathlib import Path
+
+from gammapy.maps import Map
+from gammapy.estimators import (
+    ASmoothMapEstimator,
+    TSMapEstimator,
+)
+from gammapy.irf import EDispKernelMap
+from gammapy.modeling.models import (
+    BackgroundModel,
+    SkyModel,
+    PowerLawSpectralModel,
+    PointSpatialModel,
+)
+from gammapy.irf import PSFMap, EnergyDependentTablePSF
+from gammapy.datasets import Datasets, MapDataset
+from gammapy.modeling.models import (
+    ExpCutoffPowerLawSpectralModel,
+    PointSpatialModel,
+    SkyModel,
+)
+
+
+def data_prep():
+    counts = Map.read("$GAMMAPY_DATA/fermi-3fhl-gc/fermi-3fhl-gc-counts-cube.fits.gz")
+    background = Map.read(
+        "$GAMMAPY_DATA/fermi-3fhl-gc/fermi-3fhl-gc-background-cube.fits.gz"
+    )
+    background = BackgroundModel(background, datasets_names=["fermi-3fhl-gc"])
+
+    exposure = Map.read(
+        "$GAMMAPY_DATA/fermi-3fhl-gc/fermi-3fhl-gc-exposure-cube.fits.gz"
+    )
+    # unit is not properly stored on the file. We add it manually
+    exposure.unit = "cm2s"
```

Comment:
I think we fixed this and the unit should new read correctly from the file, but please check once again, before removing...